Tools/Scripts/libraries/webkitapipy/webkitapipy/sdkdb_from_build_products.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import os
+import shutil
+import stat
+import subprocess
+import tempfile
+from typing import cast
+from pathlib import Path
+
+
+def is_executable_file(path):
```

Comment:
This is a somewhat weak implementation of it, in multiple ways. And I generally wouldn't even think of `.a` or `.dylib` as "executable".

I don't fully understand what you are trying to do, but:
- if possible, make the name more accurate;
- if it's still not 100% descriptive, add some comment that would effectively discourage people from reusing this function in new places.

Reply:
New name is `belongs_in_sdk_content`, which hopefully makes the purpose of this much clearer.

Reply:
It's better, although still seems misleading. Why don't `.a` files belong in SDK content? We used to have those in some SDK flavors.